app/assets/javascripts/components/dashboard/create_task.jsx

```diff
@@ -6,11 +6,43 @@ class TaskCreationForm extends DefaultModal {
 
   constructor(props) {
     super();
+    this.state = {
+    };
+    this.select = this.select.bind(this);
+    this.InputField = this.InputField.bind(this);
   }
 
-  print = () => {
-    console.log("The modal is closing")
-    this.closeModal();
+  InputField({ id, label, help, ...props }) {
```

Comment:
I have an "Input" component in the "helpers" folder. Would you be able to modify that component to match what you want? I think this will be good b/c then it will improve all inputs on all of our forms, and then peep can reuse this.